app/views/search/index.html.erb

```diff
@@ -4,15 +4,117 @@
 
     <p><%= t('search.summary_of_results.showing') %> <span class="l-results__emphasised t-first-record"><%= @result.first_record %></span> <%= t('search.summary_of_results.to') %> <span class="l-results__emphasised t-last-record"><%= @result.last_record %></span> <%= t('search.summary_of_results.of') %> <span class=" l-results__emphasised t-total-records"><%= @result.total_records %></span> <%= t('search.summary_of_results.firms') %></p>
 
-    <ol>
+    <ol class="l-results__list">
       <% @result.firms.each do |firm| %>
-        <li class="t-firm">
-          <span class="t-name"><%= firm.name %></span>, advisers: <%= firm.total_advisers %> (<%= firm.closest_adviser %>)
+        <li class="result t-firm">
+          <div class="result__heading-container">
+            <%= heading_tag(firm.name, level: 2, class: 'result__heading t-name') %>
+            <p class="result__adviser-distance">Adviser 1.7 miles away</p>
+          </div>
+
+          <div class="result__section result__section--first">
```

Comment:
Also consider adding `hCard` microformats to adviser contacts: http://microformats.org/wiki/hCard